src/python/pants/help/help_integration_test.py

```diff
@@ -140,8 +140,8 @@ def test_unknown_global_flags() -> None:
 def test_unknown_scoped_flags() -> None:
     pants_run = run_pants(["test", "--forcex"])
     pants_run.assert_failure()
-    assert "Unknown flag --forcex on test scope" in pants_run.stdout
-    assert "Did you mean --force" in pants_run.stdout
+    assert "Unknown flag --forcex on test scope" in pants_run.stdout, pants_run.stderr
```

Comment:
The subsequent run went green - not good! This test is flaky. This seems to imply a stdout flushing bug in pants or in the test infra for ITs.

Reply:
Yes, CI has been flaky for about two days. Stu had been trying to fix on Friday, so we haven't yet reverted. Discussed in #infra

Sounds reasonable and probably better to revert now though.

Reply:
Reverted the change to add stderr output since this was a flake. I'm looking into Console to see if it has failure-to-flush leaks by inspection on the side...

Reply:
Note that many tests are flaking, not only this one. I believe the suspected culprit is https://github.com/pantsbuild/pants/commit/0bcbfe996d2e11f22859066a1def658c2207fa19